Client/src/Components/DataCell.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Isnt this supposed to be on a new page? The upload is gonna be on the first page?

Reply:
It's just temporary for this sprint. Later on, once we have the ability to switch between pages it'll be moved to its own page.